packages/clay-card/src/Context.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import {createContext} from 'react';
+
+export interface IContext {
+	horizontal?: boolean;
```

Comment:
You can add documentation here, just to explain why this? The same for `interactive`.